tests/python/test_field.py

```diff
@@ -48,7 +47,7 @@ def test_vector_field(n, dtype, shape):
 @pytest.mark.parametrize('shape', field_shapes)
 @ti.host_arch_only
 def test_matrix_field(n, m, dtype, shape):
-    x = ti.Matrix.field(n, m, dtype=dtype, shape=shape)
+    x = ti.Matrix.field(n, m, dtype, shape)
```

Comment:
I actually think the old version is better: there are quite a few arguments to `ti.field` and making these explicit is much easier to read.

Reply:
```suggestion
    x = ti.Matrix.field(n, m, dtype=dtype, shape=shape)
```